python/python/glide/async_commands/server_modules/ft_options/ft_create_options.py

```diff
@@ -0,0 +1,527 @@
+        """
+        Get the arguments to be used for the algorithm of the vector field.
+
+        Returns:
+            List[str]: A list of arguments.
+        """
+        args = []
+        if self.dim:
+            args.extend([FtCreateKeywords.DIM, str(self.dim)])
+        if self.distanceMetric:
+            args.extend([FtCreateKeywords.DISTANCE_METRIC, self.distanceMetric.name])
+        if self.type:
+            args.extend([FtCreateKeywords.TYPE, self.type.name])
+        return args
+
+
+class VectorFieldAttributesFlat(VectorFieldAttributes):
+    """
+    Get the arguments to be used for the FLAT algorithm of the vector field.
+
+    Args:
+        dim (int): Number of dimensions in the vector.
+        distanceMetric (DistanceMetricType): The distance metric used in vector type field. Can be one of [L2 | IP | COSINE].
+        type (VectorType): Vector type. The only supported type is FLOAT32.
+        initialCap (Optional[int]): Initial vector capacity in the index affecting memory allocation size of the index. Defaults to 1024.
+    """
+
+    def __init__(
+        self,
+        dim: int,
+        distanceMetric: DistanceMetricType,
+        type: VectorType,
+        initialCap: Optional[int] = None,
+    ):
+        """
+        Initialize a new flat vector field attributes instance.
+        """
+        super().__init__(dim, distanceMetric, type)
+        self.initialCap = initialCap
+
+    def getVectorFieldAttributes(self) -> List[str]:
+        """
+        Get the arguments representing the vector field created with FLAT algorithm.
+
+        Returns:
+            List[str]: A list of FLAT algorithm type vector arguments.
+        """
+        args = super().getVectorFieldAttributes()
+        if self.initialCap:
+            args.extend([FtCreateKeywords.INITIAL_CAP, str(self.initialCap)])
+        return args
+
+
+class VectorFieldAttributesHnsw(VectorFieldAttributes):
+    """
+    Get the arguments to be used for the HNSW algorithm of the vector field.
+
+    Args:
+        dim (int): Number of dimensions in the vector.
+        distanceMetric (DistanceMetricType): The distance metric used in vector type field. Can be one of [L2 | IP | COSINE].
+        type (VectorType): Vector type. The only supported type is FLOAT32.
+        initialCap (Optional[int]): Initial vector capacity in the index affecting memory allocation size of the index. Defaults to 1024.
+        m (Optional[int]): Number of maximum allowed outgoing edges for each node in the graph in each layer. Default is 16, maximum is 512.
+        efContruction (Optional[int]): Controls the number of vectors examined during index construction. Default value is 200, Maximum value is 4096.
+        efRuntime (Optional[int]): Controls the number of vectors examined during query operations. Default value is 10, Maximum value is 4096.
+    """
+
+    def __init__(
+        self,
+        dim: int,
+        distanceMetric: DistanceMetricType,
```

Comment:
All python code uses sneak case for variable names, please update in all places
```suggestion
        distance_metric: DistanceMetricType,
```

Reply:
Updated